src/app/parcels/batch/getCenteredBatchGridDisplayColumns.tsx

```diff
@@ -140,26 +325,65 @@ const getCenteredBatchGridDisplayColumns = (
         {
             field: "packingDate",
             headerName: "Packing Date",
-            width: 150,
+            width: 200,
```

Comment:
The others are constants are they are used in multiple files. This one is only configured here. Do you think it would be clearer to have all column widths, regardless of multiple file use, in the columnWidths file?